test/gb-tracker-core.spec.ts

```diff
@@ -360,7 +360,7 @@ describe('gb-tracker-core tests', () => {
     const window = new jsdom.JSDOM(undefined, { cookieJar }).window;
 
     const gbTrackerCore = new GbTracker('testcustomer', 'area');
-    const CookiesLib = gbTrackerCore.__getInternals().COOKIES_LIB;
+    const CookiesLib = gbTrackerCore.__getInternals().COOKIES_LIB as any;
```

Comment:
The new `as any`s are a result of me making the type stronger in the main code. The legacy test code mutated the cookies library that was imported, which is no longer valid.